pkgs/top-level/all-packages.nix

```diff
@@ -30940,6 +30942,8 @@ with pkgs;
 
   ytmdl = callPackage ../tools/misc/ytmdl { };
 
+  yutto = callPackage ../tools/misc/yutto { };
```

Comment:
Couldn't we use `toPythonApplication` here as well?

Reply:
I thought yutto is an application instead of a library. Should I put it into python-packages.nix?

Reply:
Exactly